data/mods/gen3/random-teams.ts

```diff
@@ -89,7 +90,9 @@ export class RandomGen3Teams extends RandomGen4Teams {
 		case 'focuspunch':
 			return {cull: (
 				(counter.damagingMoves.length < 2 || hasMove['rest'] || counter.setupType && !hasMove['spore']) ||
-				(!hasMove['substitute'] && (counter.Physical < 4 || hasMove['fakeout']))
+				(!hasMove['substitute'] && (counter.Physical < 4 || hasMove['fakeout'])) ||
+				// Breloom likes to have coverage
+				(species.id === 'breloom' && hasMove['machpunch'] || hasMove['skyuppercut'])
```

Comment:
```suggestion
				(species.id === 'breloom' && (hasMove['machpunch'] || hasMove['skyuppercut']))
```
I'm pretty sure this is what you mean.

Reply:
i've been told in the past that putting parentheses in the last statement doesn't matter? perhaps i was subject to misinformation

Reply:
It does matter - `&&` takes precedence.
<img width="219" alt="image" src="https://user-images.githubusercontent.com/56906084/110189168-0f1c0c80-7dd3-11eb-96c2-0891667f6e81.png">